tools/ola_trigger/config.ypp

```diff
@@ -65,6 +67,12 @@ assignment :
         delete $1;
         delete $5;
       }
+    | WORD_LITERAL optional_whitespace '=' optional_whitespace INTEGER_LITERAL {
```

Comment:
Rather than repeating this change 

value:
        INTEGER_LITERAL { $$ = ola::strings::IntToString($3); } 
     | WORD_LITERAL
     | QUOTED_VALUE

Same for `argument` below.